### PR TITLE
has_many add, bug in phone.name and form for contacts new

### PR DESCRIPTION
**Pull request by techtique**
bug in app/views/contacts/_form.html.erb 
Showing /home/carlos/www/notebook/app/views/contacts/_form.html.erb where line #51 raised:

undefined local variable or method `f' for #<#<Class:0x00005621c5e97810>:0x00007fc1e9325b88>

and Showing /home/carlos/www/notebook/app/views/phones/index.html.erb where line #18 raised
undefined method `name' for nil:NilClass

### CODE REVIEW

File: app/views/contacts/_form.html.erb [hunk 1]
```diff
@@ -45,9 +45,15 @@
   <%= address_fields.select :state, options_for_states(@contact.address.state) %>
 </div>
 
-
 <% end %>
 
+<div id="phones">
+  <%= link_to_add_association('[Adicionar Telefone]', f, :phones) %>
+  <%= form.fields_for :phones do |phones_fields| %>
+    <%= render partial: "phone_fields", locals: { f: phones_fields} %>
```
Comment (jodar) [inline app/views/contacts/_form.html.erb]: De onde saiu esse `f:` ?

File: app/views/contacts/_form.html.erb [hunk 2]
```diff
@@ -45,9 +45,15 @@
   <%= address_fields.select :state, options_for_states(@contact.address.state) %>
```
Comment (jodar) [inline app/views/contacts/_form.html.erb]: Onde ta definido esse `options_for_states`?

File: app/views/contacts/_form.html.erb (hunk 1 above)
Comment (techtique) [reply app/views/contacts/_form.html.erb]: jackson criou e colocou, ta em _contacts_phones_fields

File: app/views/contacts/_form.html.erb (hunk 2 above)
Comment (techtique) [reply app/views/contacts/_form.html.erb]: build_address em contacts_controller.rb, options_for_states em helpers, application.rb

**Pull request merged**